ugvc/reports/report_wo_gt.ipynb

```diff
@@ -1235,13 +1190,61 @@
     "    af_along_genome(filtered_vcf)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def af_vs_coverage(filtered_vcf):\n",
+    "    # convert vcf to df\n",
+    "    vcf_df = vcftools.get_vcf_df(filtered_vcf)\n",
+    "    # randomly remove rows if the vcf is too large\n",
+    "    if vcf_df.shape[0]>100000:\n",
+    "        remove_n = vcf_df.shape[0]-100000\n",
+    "        drop_indices = np.random.choice(vcf_df.index, remove_n, replace=False)\n",
+    "        vcf_df = vcf_df.drop(drop_indices)\n",
+    "        \n",
+    "    if \"vaf\" in vcf_df.columns:\n",
```

Comment:
I thought that when we have VAF in the vcf we also have AF which is the same to VAF but maybe I am wrong